modules/dev/src/tests/fs.ts

```diff
@@ -0,0 +1,23 @@
+import { TestNamespace, TestObject } from "../types";
+import { fs } from "@replit/extensions";
+import { assert } from "chai";
+
+const tests: TestObject = {
+  readFile: async () => {
+    const res = await fs.readFile("package.json");
```

Comment:
i think we should read `.replit` instead, since every repl will have it

Reply:
Not all Repls have a `.replit` file

Reply:
all modern repls (including all our templates) should have one! `package.json` is rare since most of our repls are actually python

Reply:
Will fix up and structure tests better today